modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -362,7 +362,7 @@ public class TableManager implements IgniteTablesInternal, IgniteComponent {
 
     private final LowWatermark lowWatermark;
 
-    private final HybridTimestampTracker observableTimestampTracker;
+    private final HybridTimestampTrackerImpl observableTimestampTracker;
```

Comment:
let's use interface here and below. Please check this kind of changes everywhere in your patch

Reply:
I prefer to use a concrete implementation until the other is not needed.